spec/system/notifications/index_spec.rb

```diff
@@ -135,6 +140,118 @@
     end
   end
 
+  context "EmancipationChecklistReminder" do
+    context "on the first of the month with a transition-aged youth" do
+      before do
+        travel_to Time.zone.local(year, next_month, 1) do
+          sign_in volunteer
+          Rake::Task.clear
+          Casa::Application.load_tasks
+          Rake::Task["emancipation_checklist_reminder"].invoke
```

Comment:
@sandfortw we usually don't run take tasks within the specs -- can you create the records that you need using factories instead?

Reply:
@littleforest I can try! Both of these notifications are timed events--the birthday one would go out on the 15th of the month before, and the checklist one would go out on the first of the month, but I was having trouble actually triggering them without doing it manually. I can take another look though.

Reply:
@littleforest  Correct me if I am wrong, but since it is a scheduled task in the heroku scheduler, it won't be triggered by going to the correct date on local, correct? Isn't running the task manually the only way in this case?

Reply:
Ah, that makes sense. Thank you, I will try that.

Reply:
@littleforest Is this more of what you were looking for?

<img width="917" alt="image" src="https://github.com/rubyforgood/casa/assets/80081206/e9bd8809-876a-4e94-bf8b-40adaac64508">